demos/checkboxradio/product-selector.html

```diff
@@ -0,0 +1,126 @@
+			border-radius: 8em;
+		}
+		.pill, .rectangle {
+			width: 30em;
+		}
+		.border {
+			border: 2px solid #333333;
+		}
+		.toggles {
+			width: 200px;
+		}
+		.toggle-wrap, .shape {
+			display: inline-block;
+			vertical-align: top;
+		}
+		.shape-wrap {
+			margin-bottom: 2em;
+		}
+		.controls {
+			background: #ccc;
+			padding: 1em;
+			display: inline-block;
+		}
+	</style>
+</head>
+<body>
+<div class="controls">
+	<div class="brand-wrap">
+		<h2>1.) Select a brand</h2>
+		<div class="brand">
+			<label for="brand-jquery">jQuery</label>
+			<input data-background-color="#0769AD" data-color="#7ACEF4" data-background-image="url(images/jquery.png)" type="radio" name="brand" id="brand-jquery">
+			<label for="brand-jquery-ui">jQuery UI</label>
+			<input data-background-color="#B24926" data-color="#FAA523" data-background-image="url(images/jquery-ui.png)" type="radio" name="brand" id="brand-jquery-ui" checked>
+			<label for="brand-jquery-mobile">jQuery Mobile</label>
+			<input data-background-color="#108040" data-color="#3EB249" data-background-image="url(images/jquery-mobile.png)" type="radio" name="brand" id="brand-jquery-mobile">
+			<label for="brand-sizzle">Sizzle</label>
+			<input data-background-color="#9A1B1E" data-color="#FAA523" data-background-image="url(images/sizzle.png)" type="radio" name="brand" id="brand-sizzle">
+			<label for="brand-qunit">QUnit</label>
+			<input data-background-color="#390F39" data-color="#9C3493" data-background-image="url(images/qunit.png)" type="radio" name="brand" id="brand-qunit">
+		</div>
+	</div>
+	<div class="shape-wrap">
+		<h2>2.) Select a shape</h2>
+		<div class="shape-bar">
+			<label for="shape-circle">Circle</label>
+			<input type="radio" name="shape" id="shape-circle" value="circle" checked>
+			<label for="shape-square">Square</label>
+			<input type="radio" name="shape" id="shape-square" value="square">
+			<label for="shape-pill">Pill</label>
+			<input type="radio" name="shape" id="shape-pill" value="pill">
+			<label for="shape-rectangle">Rectangle</label>
+			<input type="radio" name="shape" id="shape-rectangle" value="rectangle">
+		</div>
+	</div>
+	<div class="toggle-wrap">
+		<h2>3.) customize</h2>
+		<div class="toggles">
+			<label for="color">Shadow</label>
+			<input class="toggle brand-toggle" type="checkbox" name="color" id="color">
+			<label for="border">Border</label>
+			<input class="toggle" type="checkbox" name="border" id="border">
+			<label for="background-color">Background</label>
+			<input class="toggle brand-toggle" type="checkbox" name="background-color" id="background-color" checked>
+			<label for="background-image">Background Image</label>
+			<input class="toggle brand-toggle" type="checkbox" name="background-image" id="background-image" checked>
+		</div>
+	</div>
+	<div class="shape circle background jquery-ui"></div>
+</div>
+
+<div class="demo-description">
+<p>Examples of the markup that can be used with checkboxs and radio buttons.</p>
```

Comment:
"Using two sets of radio buttons and one group of checkboxes to implement a product selector."